vtex-apps/minicart-freeshipping-bar
# Fix: better messages in Brazilian Portuguese

#### What does this PR do? \*
It creates suitable messages for Brazilian Portuguese.

#### How to test it? \*
Change browser location to pt-BR and check if messages match messages/pt-BR.json.

#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*
